File: Calculator.py
print("CALCULATOR".center(500))
print("------------------------".center(500))
value1 = float(input("Enter Your First Number: "))
value2 = float(input("Enter Your Second Number: "))

operator = input("Which operation do You want to use? (+,-,*,/): ")

if operator == "+":
    print(f"The Addition of the Number {value1} and {value2} is {value1 + value2}")
elif operator == "-":
    print(f"The Subtraction of the Number {value1} and {value2} is {value1 - value2}")
elif operator == "/":
    if value2 == 0:
         print("Cannot divide by zero")
    else:
        print(f"The Division of the Number {value1} and {value2} is {value1 / value2}")
elif operator == "*":
        print(f"The Multiplication of the Number {value1} and {value2} is {value1 * value2}")
else:
    print("Error! Operator not found")

File: calculator.cpp
#include <iostream>

using namespace std;

int main()
{

    double firstNumber, secondNumber;
    char operation;

    // firstNumber
    cout << "Enter the first number: ";
    cin >> firstNumber;

    // secondNumber
    cout << "Enter the second number: ";
    cin >> secondNumber;

    // operation
    cout << "Enter the operation you want to perform: ";
    cin >> operation;

    if (operation == '+')
    {
        cout << "The addition of the given two numbers is " << firstNumber + secondNumber << endl;
    }
    else if(operation == '-')
    {
        cout << "The subtraction of the given two numbers is " << firstNumber - secondNumber << endl;
    }
    else if(operation == '*')
    {
        cout << "The Multiplication of the given two numbers is " << firstNumber * secondNumber << endl;
    }
    else if(operation == '/')
    {
        if (secondNumber == '0')
        {
            cout << "The Division of the two numbers is " << firstNumber / secondNumber << endl;
        }
        else
        {
            cout << "Cannot divide by zero" << endl;
            return 1;
        }
    }
    else
    {
        cout << "Invalid Operation" << endl;
        return 1;
    }
    return 0;
}

